src/viam/sdk/services/mlmodel/server.cpp

```diff
@@ -104,11 +104,19 @@ ::grpc::Status MLModelServiceServer::Infer(
     }
 
     return ::grpc::Status();
-} catch (const std::exception& ex) {
-    return ::grpc::Status(grpc::INTERNAL,
-                          std::string("[Infer]: Failed with exception: ") + ex.what());
 } catch (...) {
-    return ::grpc::Status(grpc::INTERNAL, "[Infer]: Failed with an unknown exception");
+    try {
```

Comment:
I hate this. My feeling is that if you violate noexcept by throwing then you already are mandated to call `std::terminate` so why do I need to do it manually to make clang-tidy happy? I don't want every server side method to need to write this. But i'm also reluctant to disable the noexcept escape warning because there other cases (like move operators) etc where I do want the warning.

Reply:
Oof yeah, this is pretty ugly. I'm not sure what the better solution is, but sad that we're forced into this.

Reply:
Agreed. I filed https://viam.atlassian.net/browse/RSDK-3556 so we can come back to it. It may be that `noexcept` isn't worth it here if the higher level gRPC stuff can be trusted to do something semi-OK with an escaped exception. I'll add a TODO before commit.